x/interchainstaking/types/rebalance_test.go

```diff
@@ -0,0 +1,395 @@
+package types_test
+
+import (
+	"sort"
+	"testing"
+
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ingenuity-build/quicksilver/utils/addressutils"
+	"github.com/ingenuity-build/quicksilver/x/interchainstaking/types"
+	"github.com/stretchr/testify/require"
+)
+
+func GenerateDeterministicValidators(n int) (out []string) {
```

Comment:
```suggestion
func GenerateValidatorsDeterministic(n int) (out []string) {
```

I think this name is clearer